go/utils/restclient.go

```diff
@@ -17,8 +17,10 @@ import (
 // HTTPOK is ok status of http api call.
 const HTTPOK = "200 OK"
 
-var HttpClient = &http.Client{Transport: &http.Transport{}}
+// HTTPClient is a global Http client obj conains one persist connection.
+var HTTPClient = &http.Client{Transport: &http.Transport{}}
```

Comment:
Just curious, `http.DefaultClient` seems a reasonable starting point, what is the benefit of using `var HTTPClient = &http.Client{Transport: &http.Transport{}}`?